test/support/factory.ex

```diff
@@ -1,5 +1,6 @@
 defmodule PointingParty.Factory do
-  use ExMachina.Ecto, repo: PointingParty.Repo
+  use ExMachina
```

Comment:
We probably don't even need ExMachina now.

Reply:
I agree. If we realize we need it later, we can always add it back. We may not even need `PointingPart.Factory`.